### ISSUE TITLE
More Field Gun feedback (they dont have any elevation/depression, meaning their cannon is stuck)

### ISSUE THREAD

**Issue by Firedow** [spring1944/spring1944#451]
It seems that the field guns we're using can't move their actual cannon up and down, leading them to have no elevation/depression whatsoever, this leads to awkwards situations in which the AT gun that is on high ground, cant aim down its cannon so it has to slowly walk down the hill in order to shoot in the same height
and Howitzers that would be in high ground, cant aim down to adjust their fire and therefore won't fire on different heights, only heights that are *possible within their arc* in which the cannon is forever stuck to

**Comment by sanguinariojoe** [comment]
Yeah, the problem is that in Spring you cannot have different settings for horizontal angle tolerance and depression. So probably I shall find some sort of balance
